docs/UPGRADING.md

```diff
@@ -2,6 +2,394 @@
 
 Here you can find a list of migration guides to handle breaking changes between releases of the CLI.
 
+## 0.18.0
```

Comment:
This must be `## Unreleased`, it will be changed when preparing for the new release.